src/web/app/pages-admin/admin-sessions-page/admin-sessions-page.component.ts

```diff
@@ -85,6 +101,31 @@ export class AdminSessionsPageComponent implements OnInit {
 
     this.getFeedbackSessions();
   }
+  num: number=0;
+  /**
+   * Populates the Sortable Table Data to be displayed
+   */
+  populateSortableTable(): void {
+  this.sortableTables=[];
+  this.num=0;
+  const key= Object.keys(this.sessions)
+    Object.values(this.sessions).forEach((ongoingSessionModelArray)=>{ 
+      this.sortableTables.push({
+        columns: this.column ,
+        rows: ongoingSessionModelArray.map((session): SortableTableCellData[]=>{
+          return [
+            { displayValue: session.ongoingSession.sessionStatus },
+            { displayValue: '['+session.ongoingSession.courseId+'] '+session.ongoingSession.feedbackSessionName },
+            { displayValue: ''},
+            { displayValue: session.startTimeString },
+            { displayValue: session.endTimeString },
```

Comment:
I think the issue lies here! You need to fill in the `value` field which is used for sorting.

Reply:
You're right, thanks!